### ISSUE TITLE
[Bug] "The media could not be loaded, either because the server or network failed or because the format is not supported."

### ISSUE THREAD

**Issue by DUOLabs333** [iv-org/invidious#4947]
**Describe the bug**
For _almost_ all videos (shorts and a few other videos seem to be immune), I started getting this error: "The media could not be loaded, either because the server or network failed or because the format is not supported." This started about 2 days ago.

**Steps to Reproduce**
1. Go to https://<invidious instance>/watch?v=tPTvG0Se7L8
2. See error

**Logs**

**Screenshots**

**Additional context**
 - Browser (if applicable): Chrome 120
 - OS (if applicable): macOS 11

I tried adding the `po_token` and `visitor_data` from https://github.com/iv-org/youtube-trusted-session-generator, but it didn't help.

Switching to `master-arm64` also didn't help.

**Comment by UltraBlackLinux** [comment]
wtf how can an old issue be a duplicate of a new issue??

The above fix doesn't appear to be working for me. I don't know how I'd check if the sig helper is actually doing anything however. I do see some `sig` and `lsig` tags in the requests to google, though no idea if that's relevant.

I have tried clearing out the cached videos, but 
```
postgres@9ff3c0e5b3f8:/$ psql invidious -c "TRUNCATE TABLE videos"
psql: error: connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: FATAL:  role "postgres" does not exist
```
(no idea what to make of that either :/ )

**Comment by unixfox** [comment]
> Also fixed the problem for me (also for previously failed videos); also did not know that this was needed. It's in the [install docs](https://docs.invidious.io/installation/#docker-compose-method-production) already, but I didn't know it changed/had an update.
> 
> Some sort of notification for new install requirements would be cool!
> 
> Thanks for the info/fix :)

Invidious has releases now! With a changelog of what changed: https://github.com/iv-org/invidious/releases/tag/v2.20240825

So there were technically an announcement back in the end of august.

**Comment by thebigbone** [comment]
After adding `po_token` and `visitor_data`, I am getting this error:
```
audio append of 61410b failed for segment #0 in playlist 0-placeholder-uri-AUDIO-audio-main
```

**Comment by gigq** [comment]
Getting the same error, installing inv_sig_helper and potoken haven't made a difference. I'm running outside of docker if that matter but have all the requirements mentioned in this thread running but still get the "The media could not be loaded, either because the server or network failed or because the format is not supported."

**Comment by DUOLabs333** [comment]
@gigq Make sure the signature_helper key is in the tank config, and that you truncated the videos database.

**Comment by red0bear** [comment]
In reality i cant watch any video anymore on using this program because of "The media could not be loaded, either because the server or network failed or because the format is not supported." cant listen audio/ download too . This error started last week here. I see youtube_dl cant be used too this time to get videos.

**Comment by thebigbone** [comment]
If anyone wants to keep rotating the tokens, I wrote this small script which you can run it in a cronjob:
```sh
output=$(docker run quay.io/invidious/youtube-trusted-session-generator)

visitor_data=$(echo "$output" | awk -F': ' '/visitor_data/ {print $2}')
po_token=$(echo "$output" | awk -F': ' '/po_token/ {print $2}')

sed -i "s/visitor_data:.*/visitor_data: $visitor_data/" docker-compose.yml
sed -i "s/po_token:.*/po_token: $po_token/" docker-compose.yml

docker compose up -d invidious
```

**Comment by techmetx11** [comment]
> > @gigq Make sure the signature_helper key is in the tank config, and that you truncated the videos database.
> 
> Thank you that worked. For anyone else running outside docker these are the additional steps:
> 
> Pull the latest invidious, recompile.
> 
> Generate po_token and visitor_data identities (https://docs.invidious.io/installation/#generate-po_token-and-visitor_data-identities)
> 
> Add the po_token and visitor_data to your config.yml
> 
> Build and run inv_sig_helper (https://docs.invidious.io/installation/#run-inv_sig_helper-in-background)
> 
> Add signature_server to your config.yml If you are running it locally this should be "signature_server: 127.0.0.1:12999" however I also had to specify this at start up when running inv_sig_helper even though that port is supposed to be the default: inv_sig_helper_rust --tcp 127.0.0.1:12999
> 
> Anyways this got me back up and running hope it helps others.

If you're running it locally (and outside of Docker), you should use a UNIX socket instead, since it'll be more performant and usable than TCP

**Comment by thebozzcl** [comment]
Confirmed working after adding the signature helper! Now I need to find a setup that can rotate the trusted sessions automatically within Docker Swarm.

**Comment by MenacingMight** [comment]
Hello I added the signature helper but most videos still don't work and I get `403 GET /videoplayback` error. Weirdly enough I found that MKBHD videos work with a `206 GET /videoplayback` status but anything else does not.

Signature helper at log startup
```
inv_sig_helper-1  | [2024-09-26T00:00:29Z INFO  inv_sig_helper_rust] Fetching player
inv_sig_helper-1  | [2024-09-26T00:00:30Z INFO  inv_sig_helper_rust::player] Fetching player JS URL: https://www.youtube.com/s/player/c9dd45ed/player_ias.vflset/en_US/base.js
inv_sig_helper-1  | [2024-09-26T00:00:30Z WARN  inv_sig_helper_rust::player] nsig function ending did not work: =\s*function(\([\w]+\)\{\s*var\s+[\w\s]+=[\w\.\s]+?\.call\s*\([\w\s$]+?,[\(\)\",\s]+\)[\S\s]*?\}\s*return [\w\.\s$]+?\.call\s*\([\w\s$]+?\s*,[\(\)\",\s]+\)\s*\}\s*;)
inv_sig_helper-1  | [2024-09-26T00:00:30Z INFO  inv_sig_helper_rust::player] sig code: var iCa;var iL={Z2:function(a,b){a.splice(0,b)},
inv_sig_helper-1  |     h5:function(a){a.reverse()},
inv_sig_helper-1  |     YZ:function(a,b){var c=a[0];a[0]=a[b%a.length];a[b%a.length]=c}};iCa=function(a){a=a.split("");iL.h5(a,0);iL.Z2(a,3);iL.YZ(a,4);iL.YZ(a,25);iL.h5(a,21);iL.Z2(a,2);iL.YZ(a,17);iL.YZ(a,29);return a.join("")}
inv_sig_helper-1  | [2024-09-26T00:00:30Z INFO  inv_sig_helper_rust] Successfully fetched player
```

**Comment by DUOLabs333** [comment]
@MenacingMight Did you add the signature_helper to your yaml?

**Comment by cleancoda** [comment]
Adding to the thread - went through the process of running the session generator via ``docker run quay.io/invidious/youtube-trusted-session-generator``, got the two keys 
- po_token
- visitor_data identities

plugged them into docker-compose.yml and added those two new keys, along with sig helper and adding ``signature_server: inv_sig_helper:12999`` as well.

``
inv_sig_helper:
    image: quay.io/invidious/inv-sig-helper:latest
    command: ["--tcp", "0.0.0.0:12999"]
    environment:
      - RUST_LOG=info
    restart: unless-stopped
    cap_drop:
      - ALL
    read_only: true
    security_opt:
      - no-new-privileges:true
``

observations:

1. If no account is logged into the instance, every video plays.
2. if you're logged into an existing account with subs/playlists, no video plays.
3. if you create a brand new account, with no subs or playlists, every video plays.
4. if you export invidious settings from import/export under settings, and import into the new account you just created, no video plays.

3. and 4. have thrown me off in a loop. Not sure what is causing that. Any clues on what could be the reason? 

Thank you for all your hardwork. we shall prevail! :)

**Comment by MenacingMight** [comment]
@DUOLabs333 yes I have it in my docker compose yaml. I even tried using the exact same compose file in the documentation and it's still the same. I also tried using a fresh backend but it didn't do anything.

**Comment by MSWS** [comment]
> If anyone wants to keep rotating the tokens, I wrote this small script which you can run in a cronjob:
> 
> ```shell
> output=$(docker run quay.io/invidious/youtube-trusted-session-generator)
> 
> visitor_data=$(echo "$output" | awk -F': ' '/visitor_data/ {print $2}')
> po_token=$(echo "$output" | awk -F': ' '/po_token/ {print $2}')
> 
> sed -i "s/visitor_data:.*/visitor_data: $visitor_data/" docker-compose.yml
> sed -i "s/po_token:.*/po_token: $po_token/" docker-compose.yml
> 
> docker compose up -d invidious
> ```

A small enhancement I made to this script allows for specifying these variables in a `.env` instead of in the compose directly. Also added the `--rm` flag so we don't create a dangling container every time.

```sh
#!/usr/bin/env bash

output=$(docker run --rm quay.io/invidious/youtube-trusted-session-generator)

visitor_data=$(echo "$output" | awk -F': ' '/visitor_data/ {print $2}')
po_token=$(echo "$output" | awk -F': ' '/po_token/ {print $2}')

sed -i "s/VISITOR_DATA=.*/VISITOR_DATA=$visitor_data/" [PATH]/.env
sed -i "s/PO_TOKEN=.*/PO_TOKEN=$po_token/" [PATH]/.env

docker compose -f /home/iboaz/invidious/compose.yml up --force-recreate -d
```

**Comment by DX37** [comment]
Any idea how to combine this inv_sig_helper with gluetun?

**Comment by porchthecoder** [comment]
Have the same 403 issue as everyone else with an docker install. Started a few days ago. Added inv-sig-helper and that gets Medium/HD720/Small working with but not DASH. Proxy videos has no effect.

**Comment by cpumaxx** [comment]
I'm having the same issue, and Debian stable doesn't have new enough rust to build inv-sig-helper. Is there another workaround?

**Comment by kkwpsi** [comment]
Install rust & cargo locally:

> curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh

**Comment by red0bear** [comment]
after try add inv_sig and run it on docker still have getting same error.

**Comment by Bubba8291** [comment]
> Have the same 403 issue as everyone else with an docker install. Started a few days ago. Added inv-sig-helper and that gets Medium/HD720/Small working with but not DASH. Proxy videos has no effect.

@porchthecoder Same issue. But if I set my account default to DASH, it gives the format no supported error. Setting the default to medium will play all videos at medium.

**Comment by UltraBlackLinux** [comment]
Logging out of my account within invidious didn't help the problem in my case.
And I doubt it'd block my IP since my ISP rotates them daily - unless they're linking that up with the token stuff...
Unlike for some other people here, changing the quality settings hasn't worked out for me either. Neither has setting up the tokens and the sig helper. 
Quite unfortunate really...

**Comment by tcsenpai** [comment]
Same error here in docker.
po_token  and visitor_data are set in the yml and so is the signature_server.

I tested different compose configurations (included with and without the po_token/visitor_data , rotating them too) and I still get:

`2024-09-28 10:25:42 UTC [info] 403 GET /videoplayback?expire=1727540741&ei=pdn3ZrbODIKP6dsPk82eyAs&ip=87.121.148.217&id=o-APGzA0C1oqM4PkDVl4OIu0Cd7EQMvvmDXAwPLUSRuVWm&itag=396&aitags=133%2C134%2C135%2C136%2C137%2C160%2C242%2C243%2C244%2C247%2C248%2C278%2C394%2C395%2C396%2C397%2C398%2C399&source=youtube&requiressl=yes&xpc=EgVo2aDSNQ%3D%3D&mh=hP&mm=31%2C26&mn=sn-hpa7knle%2Csn-4g5edndd&ms=au%2Conr&mv=m&mvi=1&pl=23&initcwndbps=2930000&bui=AXLXGFR8-GTFkbdfC82uOk4TZut_1hbJy9Jw8HQgFQSWjM94zk29xOthtGQ4tizya4fcnZW8biDcUpcb&vprv=1&svpuc=1&mime=video%2Fmp4&ns=yWhK_XJle9gFgDx-3FCP0GEQ&rqh=1&gir=yes&clen=7257892&ratebypass=yes&dur=520.520&lmt=1727365109973585&mt=1727518619&fvip=4&keepalive=yes&fexp=51299152&c=TVHTML5_SIMPLY_EMBEDDED_PLAYER&sefc=1&txp=5537434&n=YeseYD5qhYc8IwsxR&sparams=expire%2Cei%2Cip%2Cid%2Caitags%2Csource%2Crequiressl%2Cxpc%2Cbui%2Cvprv%2Csvpuc%2Cmime%2Cns%2Crqh%2Cgir%2Cclen%2Cratebypass%2Cdur%2Clmt&sig=AJfQdSswRgIhAN73IRFTvQOWPvpcpSwzAWkxAvmtKHw0qAE9w0spqpD_AiEAq-0feNGsWh4hCrXwfvp97-Pt8AgKtF17yGlVaEHa2TA%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=ABPmVW0wRQIgCrPPsoMTU0Bh9MeL0A4ujdXZlgJayRB3-PR-16ICWPgCIQC64Au3nR6FdP3K7pf1MgrUXakkGzWMVAsi-3BlpxrT4w%3D%3D&host=rr1---sn-hpa7knle.googlevideo.com 204.29ms`

**Comment by Berkopl** [comment]
Hi any solution for the killer bug?

**Comment by MenacingMight** [comment]
> Hello I added the signature helper but most videos still don't work and I get `403 GET /videoplayback` error. Weirdly enough I found that MKBHD videos work with a `206 GET /videoplayback` status but anything else does not.
> 
> Signature helper at log startup
> 
> ```
> inv_sig_helper-1  | [2024-09-26T00:00:29Z INFO  inv_sig_helper_rust] Fetching player
> inv_sig_helper-1  | [2024-09-26T00:00:30Z INFO  inv_sig_helper_rust::player] Fetching player JS URL: https://www.youtube.com/s/player/c9dd45ed/player_ias.vflset/en_US/base.js
> inv_sig_helper-1  | [2024-09-26T00:00:30Z WARN  inv_sig_helper_rust::player] nsig function ending did not work: =\s*function(\([\w]+\)\{\s*var\s+[\w\s]+=[\w\.\s]+?\.call\s*\([\w\s$]+?,[\(\)\",\s]+\)[\S\s]*?\}\s*return [\w\.\s$]+?\.call\s*\([\w\s$]+?\s*,[\(\)\",\s]+\)\s*\}\s*;)
> inv_sig_helper-1  | [2024-09-26T00:00:30Z INFO  inv_sig_helper_rust::player] sig code: var iCa;var iL={Z2:function(a,b){a.splice(0,b)},
> inv_sig_helper-1  |     h5:function(a){a.reverse()},
> inv_sig_helper-1  |     YZ:function(a,b){var c=a[0];a[0]=a[b%a.length];a[b%a.length]=c}};iCa=function(a){a=a.split("");iL.h5(a,0);iL.Z2(a,3);iL.YZ(a,4);iL.YZ(a,25);iL.h5(a,21);iL.Z2(a,2);iL.YZ(a,17);iL.YZ(a,29);return a.join("")}
> inv_sig_helper-1  | [2024-09-26T00:00:30Z INFO  inv_sig_helper_rust] Successfully fetched player
> ```

Update
I usually run the invidious container on Unraid but since it wasn't working I used the updated docker compose yaml in a vm and now it works. Its the same compose file with the same db data. The only thing that is different is where its running. Anyone have any ideas on how to debug this further and why running it Unraid would cause problems?

**Comment by Lucky-Lobo** [comment]
Hey,

I just commented out the `po_token` and `visitor_data` in my docker compose file - and somehow it seems to be working as expected now.

**Comment by tcsenpai** [comment]
I am not sure why this is closed or marked as complete @unixfox .
As per my comment [here](https://github.com/iv-org/invidious/issues/4947#issuecomment-2380595246) for example, I have po_token and visitor_data set but the error still persists.

**Comment by unixfox** [comment]
@tcsenpai If you have thoroughly follow the official guide, **which includes generating a po_token, visitor_data AND configuring the inv_sig_helper** like explained in https://docs.invidious.io/installation/

Then I'm sorry for the moment there is nothing else that you can do. Your IP has probably been blocked by YouTube.

**Comment by tcsenpai** [comment]
> @tcsenpai If you have thoroughly followed the official guide, **which includes generating a po_token, visitor_data AND configuring the inv_sig_helper** like explained in https://docs.invidious.io/installation/
> 
> Then I'm sorry for the moment there is nothing else that you can do. Your IP has probably been blocked by YouTube.

Oh ok, sad to hear that. I tried with every IP could try with (one domestic and the others with a VPN) and I am still unable to have a working video. Yes I configured inv_sig_helper in the docker and generated the token and data indeed.

Weird enough, with the same VPN and IP I am using in gluetun i can access the normal youtube frontend just fine.

**Comment by red0bear** [comment]
> > @tcsenpai If you have thoroughly followed the official guide, **which includes generating a po_token, visitor_data AND configuring the inv_sig_helper** like explained in https://docs.invidious.io/installation/
> > Then I'm sorry for the moment there is nothing else that you can do. Your IP has probably been blocked by YouTube.
> 
> Oh ok, sad to hear that. I tried with every IP could try with (one domestic and the others with a VPN) and I am still unable to have a working video. Yes I configured inv_sig_helper in the docker and generated the token and data indeed.
> 
> Weird enough, with the same VPN and IP I am using in gluetun i can access the normal youtube frontend just fine.
> 
> UPDATE: I moved both gluetun and invidious to another home server I have (same model , same network, same ip as the other). Now everything works.
> 
> _**NOTE**_: To use gluetun and the sig_helper container together on the new server I had to assign to the sig_helper container in the docker-compose.yml file a gluetun network address and edit it above.
> 
> Example:
> 
> In docker-compose.yml
> 
> ```
> invidious:
>     [...]
>     environment:
>     [...]
>       signature_server: 172.80.0.32
> 
> [....]
> 
> inv_sig_helper:
>     [...]
>     networks:
>       gluetun_network:
>         172.80.0.32
> [....]
> ```

may be they are blocking by MAC ADDRESS ?

**Comment by tcsenpai** [comment]
> > > @tcsenpai If you have thoroughly followed the official guide, **which includes generating a po_token, visitor_data AND configuring the inv_sig_helper** like explained in https://docs.invidious.io/installation/
> > > Then I'm sorry for the moment there is nothing else that you can do. Your IP has probably been blocked by YouTube.
> > 
> > 
> > Oh ok, sad to hear that. I tried with every IP could try with (one domestic and the others with a VPN) and I am still unable to have a working video. Yes I configured inv_sig_helper in the docker and generated the token and data indeed.
> > Weird enough, with the same VPN and IP I am using in gluetun i can access the normal youtube frontend just fine.
> > UPDATE: I moved both gluetun and invidious to another home server I have (same model , same network, same ip as the other). Now everything works.
> > _**NOTE**_: To use gluetun and the sig_helper container together on the new server I had to assign to the sig_helper container in the docker-compose.yml file a gluetun network address and edit it above.
> > Example:
> > In docker-compose.yml
> > ```
> > invidious:
> >     [...]
> >     environment:
> >     [...]
> >       signature_server: 172.80.0.32
> > 
> > [....]
> > 
> > inv_sig_helper:
> >     [...]
> >     networks:
> >       gluetun_network:
> >         172.80.0.32
> > [....]
> > ```
> 
> may be they are blocking by MAC ADDRESS ?


I don't think so because both my servers are behind the same router (which also has a static IP), thus their mac addresses shouldn't hit the web.

I would blame some sort of reused resource by docker but I am not enough proficient to elaborate more

**Comment by red0bear** [comment]
> > > > @tcsenpai If you have thoroughly followed the official guide, **which includes generating a po_token, visitor_data AND configuring the inv_sig_helper** like explained in https://docs.invidious.io/installation/
> > > > Then I'm sorry for the moment there is nothing else that you can do. Your IP has probably been blocked by YouTube.
> > > 
> > > 
> > > Oh ok, sad to hear that. I tried with every IP could try with (one domestic and the others with a VPN) and I am still unable to have a working video. Yes I configured inv_sig_helper in the docker and generated the token and data indeed.
> > > Weird enough, with the same VPN and IP I am using in gluetun i can access the normal youtube frontend just fine.
> > > UPDATE: I moved both gluetun and invidious to another home server I have (same model , same network, same ip as the other). Now everything works.
> > > _**NOTE**_: To use gluetun and the sig_helper container together on the new server I had to assign to the sig_helper container in the docker-compose.yml file a gluetun network address and edit it above.
> > > Example:
> > > In docker-compose.yml
> > > ```
> > > invidious:
> > >     [...]
> > >     environment:
> > >     [...]
> > >       signature_server: 172.80.0.32
> > > 
> > > [....]
> > > 
> > > inv_sig_helper:
> > >     [...]
> > >     networks:
> > >       gluetun_network:
> > >         172.80.0.32
> > > [....]
> > > ```
> > 
> > 
> > may be they are blocking by MAC ADDRESS ?
> 
> I don't think so because both my servers are behind the same router (which also has a static IP), thus their mac addresses shouldn't hit the web.
> 
> I would blame some sort of reused resource by docker but I am not enough proficient to elaborate more

Deleted all containers and recompiled/installed the stuff again same error.

**Comment by sainttttt** [comment]
> > Oh, I think it does work, but cached videos (even those that were opened unsuccessfully) are not sent to the helper.
> 
> Yeah, truncation was needed (I remembered the command --- `psql -d invidious -c 'TRUNCATE TABLE videos'`)
> 
> > Are you managing your containers in Portainer?
> 
> No, I use my own scripts.

In case anyone is using docker and this can help, I used this command to truncate the videos table in docker which allowed me to view previously failed videos:

```
 docker exec invidious-invidious-db-1 psql -U kemal -d invidious -c 'TRUNCATE TABLE videos'
```

**Comment by eleaner** [comment]
> Adding to the thread - went through the process of running the session generator via `docker run quay.io/invidious/youtube-trusted-session-generator`, got the two keys
> 
> * po_token
> * visitor_data identities
> 
> plugged them into docker-compose.yml and added those two new keys, along with sig helper and adding `signature_server: inv_sig_helper:12999` as well.
> 
> `inv_sig_helper: image: quay.io/invidious/inv-sig-helper:latest command: ["--tcp", "0.0.0.0:12999"] environment: - RUST_LOG=info restart: unless-stopped cap_drop: - ALL read_only: true security_opt: - no-new-privileges:true`
> 
> observations:
> 
> 1. If no account is logged into the instance, **every video plays**.
> 2. if you're logged into an existing account with subs/playlists, **no video plays**.
> 3. if you create a brand new account, with no subs or playlists, **every video plays**.
> 4. if you export invidious settings from import/export under settings, and import into the new account you just created, **no video plays**.
> 
> items # 3. and 4. have thrown me off in a loop. Not sure what is causing that. Any clues on what could be the reason?
> 
> Thank you all for your incredible hard work!! we shall prevail!! :)

did anyone ever answer that?

**Comment by Lucky-Lobo** [comment]
Funny, something must have changed, it all worked perfectly fine until last night. I started getting 403 when watching on DASH - switching to HD720 it's all 206.
Tried to add `po_token` and `visitor_data` but, also the truncation several times but no such luck.

Is anyone else having these issues?

**Comment by need4swede** [comment]
Also having issues

**Comment by pflavio** [comment]
Same for me, only 720p works (DASH doens't work at all regardless of quality setting).

**Comment by danvolchek** [comment]
Seeing the same behavior now - even after the addition of the sig helper resolved the first issue. I think this is a new one/needs a new issue.

**Comment by Lucky-Lobo** [comment]
> Seeing the same behavior now - even after the addition of the sig helper resolved the first issue. I think this is a new one/needs a new issue.

I do have the sig-helper running, sill the issue, so I'm not quite sure if the problem is there.

![image](https://github.com/user-attachments/assets/d3b0bb61-7a34-4576-a419-b527558b2e52)

**Comment by unixfox** [comment]
No. The original issue is https://github.com/iv-org/invidious/issues/4734

**Comment by unixfox** [comment]
We have created a dedicated page for the various issues related to "The media could not be loaded, either because the server or network failed or because the format is not supported.": http://127.0.0.1:8000/youtube-errors-explained/#videoplayback-urls-that-returns-403-http-errors

If that didn't solve your issue then please diagnose your issue yourself before opening a bug report: https://docs.invidious.io/devtools/

**Comment by techmetx11** [comment]
> http://127.0.0.1:8000/youtube-errors-explained/#videoplayback-urls-that-returns-403-http-errors

Very informative page /s

**Comment by bluuurgh** [comment]
> > http://127.0.0.1:8000/youtube-errors-explained/#videoplayback-urls-that-returns-403-http-errors
> 
> Very informative page /s

https://docs.invidious.io/youtube-errors-explained/#videoplayback-urls-that-returns-403-http-errors

**Comment by Lucky-Lobo** [comment]
Hi, 

it's been a while and i thought i give it another shot - but somehow the issue persists.

I have the `.har ` file attatched, hope any of you guys can find something. 

[lab.har.zip](https://github.com/user-attachments/files/18645290/lab.har.zip)

**Comment by dreamos82** [comment]
I just read the documentation here: https://docs.invidious.io/youtube-errors-explained/#videoplayback-urls-that-returns-403-http-errors 

But i don't think it lists all the cases. Since it mentions only IP or location problems. 

I Got this on my dev tools: 

![Image](https://github.com/user-attachments/assets/31c75941-4a70-4eab-b2e7-957889bbe48d)

This is the URL request generated: 
```
/videoplayback?expire=1751475395&ei=YxBlaPKZA5rR9fgP9diCgAk&ip=51.187.193.207&id=o-AB-PfFVwFnZgiz7U8CWvX64CAq3c0ok2uaDpOA7vbUKW&itag=398&aitags=133%2C134%2C135%2C136%2C160%2C242%2C243%2C244%2C247%2C278%2C298%2C299%2C302%2C303%2C308%2C394%2C395%2C396%2C397%2C398%2C399%2C400&source=youtube&requiressl=yes&xpc=EgVo2aDSNQ%3D%3D&met=1751453795%2C&mh=L1&mm=31%2C29&mn=sn-cn3tc-aigy%2Csn-q0c7rn76&ms=au%2Crdu&mv=m&mvi=3&pl=22&rms=au%2Cau&initcwndbps=3825000&bui=AY1jyLMwQ9kSL7kiiUfK5HEScbc8RFL1ViSDmIN0PqkHAsfiYxWVKd29ugwsqnnZKbTozlmJGH_P6sf6&spc=l3OVKT_5L82XBn-bPjYiF_9xP_eAHmVGujnbA_YePFfB69Y-VGXjqtllWJJ7ChbmFw&vprv=1&svpuc=1&mime=video%2Fmp4&ns=VrE1uLRrNEzFPubXA4CJpc4Q&rqh=1&gir=yes&clen=120463727&ratebypass=yes&dur=810.626&lmt=1651483574551658&mt=1751453351&fvip=4&keepalive=yes&lmw=1&c=TVHTML5&sefc=1&txp=4532434&n=nn9rhdmc6ywU1FEEYg&sparams=expire%2Cei%2Cip%2Cid%2Caitags%2Csource%2Crequiressl%2Cxpc%2Cbui%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cns%2Crqh%2Cgir%2Cclen%2Cratebypass%2Cdur%2Clmt&sig=AJfQdSswRAIgdxnpM2-arVYEj6Y8rUDGUIiaeYMCxDoHPrXx-XRDRRsCIGvnz502NJRfbMZvp1ceDITnZWDc3SH0Yxab2Mg3y0QS&lsparams=met%2Cmh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Crms%2Cinitcwndbps&lsig=APaTxxMwRgIhAK0ljydcK_-i23MhLrv7fTvVVcfL-zmVhQNSJmvGzmQ1AiEA49S6XguLbe6yW-9rShguUhWTAsd3lRNoradk9LIO-EE%3D&pot=MpQBuIADnMoVyda4uFSetKhhIdLPbDU470jxzT5izJiaAHOW0KZ3F80WbHObO_-wtHTQdDm6yGQt0itNckjxnz-e5fl_jOJTts8TTC36Apb-gNcjLEG4QJvEioFpSb4Vn7VnbAITz4Dj9wTF8oWlurvOqPXKgRdeuGjk6PxN6aq_2sNZy4zUjpa2eBcl8wtcH5nWt2vYMQ%3D%3D&host=rr3---sn-cn3tc-aigy.googlevideo.com 252.57ms
```

Now if my IP has been banned, or there are some geographic restrition on the video, I shouldn't be able to see the same video on the same machine with the same IP using youtube directly, right? 

The videos loads perfectly for me on youtube. But using invidious i got this error (anyway tried with few different videos and i get the same error). Any suggestions/help is welcome. 

I can see the channel, do searches, and view other stuff.
